src/nvim/misc1.c

```diff
@@ -2927,12 +2930,11 @@ char_u *vim_getenv(char_u *name, int *mustfree)
     return p;
   }
 
-  vimruntime = (STRCMP(name, "VIMRUNTIME") == 0);
```

Comment:
That is why I was asking, but I assumed I might be missing something so I left the comparisons exactly as I found them for fear of breaking something else.